torch/csrc/jit/tensorexpr/operators/misc.cpp

```diff
@@ -642,6 +642,97 @@ Tensor computeCat(
       });
 }
 
+static bool checkStackInputShape(const std::vector<BufHandle>& bufList) {
+  if (bufList.size() == 0) {
+    throw std::runtime_error("Empty input list is passed to aten::stack");
```

Comment:
Nit: let's be consistent in handling error cases. Two lines below we use `TORCH_INTERNAL_ASSERT` and here we're throwing an error.